homepairs/homepairsUI/src/Screens/Components/HomePairsHeader/HomePairsHeaderMenu.tsx

```diff
@@ -163,6 +164,7 @@ export default class HomePairsMenu extends React.Component<Props> {
         // to the Properties
         this.setSelected(value);
         this.closeMenu();
+        console.log(setAuthenticatedState)
```

Comment:
Try to get rid of console.log statements for development please! Granted, if you plan on working with HomePairsHeader a lot this next sprint, and you would remember to get rid of without reminder, then feel free to keep it. But probably best to get rid of it and re-add it next time you need it.